server/templates/static/homepage.html

```diff
@@ -90,6 +90,23 @@ <h2>Explore the data</h2>
   </div>
 </section>
 
+<section id="by-the-numbers">
+  <div class="container">
+    <div class="highlight">
+      <h3>By the numbers</h3>
+      <p>We have continuously added data to the Data Commons graph over the last 5 years. As of Jan 2022, the graph contains:</p>
+      <ul>
+        <li>515 B triples</li>
```

Comment:
should we expand - B, M, K - to make it clear?

Reply:
Perhaps `2.8 billion`, `2.8 million`, `90,000`, `200,000` ...

Reply:
yea that's a good suggestion!